docs/src/components/theme-designer/contrast-level.astro

```diff
@@ -0,0 +1,86 @@
+---
+export interface Props {
+	labels: {
+		label: string;
+	};
+}
+const { labels = { label: 'WCAG Contrast Level' } } = Astro.props;
+---
+
+<contrast-level-toggle class="sl-flex">
+	<fieldset class="not-content">
+		<legend>{labels.label}</legend>
```

Comment:
This is pretty technical, should we link somehow to a reference or something describing what `AA` & `AAA` are?

Reply:
Didn’t find many amazing resources. Maybe the best link I dug up so far is the [MDN colour contrast guide](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast).

Not quite sure how best to integrate it though.

One option is a sentence in the intro text:

![theme editor and preamble with a sentence reading "Use the Contrast Level option to specify which of the Web Content Accessibility Guideline colour contrast standards to meet"](https://github.com/user-attachments/assets/87483025-29e8-4ce4-8b13-af23f39bda5e)

Or less intrusive could be some kind of icon link in the group label itself, but also not super clear what the icon is in this case:

![theme editor with an information icon next to the "WCAG Contrast Level" label](https://github.com/user-attachments/assets/f232b4d9-3941-47fb-9f8b-94a3543fa131)

Reply:
Alternatively could also label things with `4.5:1` / `7:1` although not sure that’s any _more_ helpful :Swe

Reply:
Or as another option:

![contrast level picker with two options: "Minimum (AA)" and "Enhanced (AAA)"](https://github.com/user-attachments/assets/cfad1d53-b3e1-4ed4-b9ec-7fa71fa6833a)

Reply:
Thanks for trying and sharing all the options 🙌 

As simple as it is, I think personally prefer the first option with a sentence and a link (and indeed, the MDN one seems like the best to link to).

Reply:
Yeah, I think I do too. Added! Thanks for the helpful feedback.